src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -4079,5 +4079,25 @@ private bool notifyInstructionSetUsage(InstructionSet instructionSet, bool suppo
             return supportEnabled ? _compilation.InstructionSetSupport.IsInstructionSetSupported(instructionSet) : false;
         }
 #endif
+
+        private static bool tryReadRvaFieldData(FieldDesc field, byte* buffer, int bufferSize, int valueOffset)
```

Comment:
```suggestion
        private static bool TryReadRvaFieldData(FieldDesc field, byte* buffer, int bufferSize, int valueOffset)
```
We can use standard naming convention for internal details. (The non-standard naming convention is only used for methods on JIT/EE interface.)